channel-manager-operations/availabilityBlock.md

```diff
@@ -130,36 +130,44 @@ The example shows a valid availability block definition.
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `start` | `string` | required | Start date in format `"yyyy-MM-dd"` \(e.g. `"2021-12-24"`\). |
-| `end` | `string` | required | End date in format `"yyyy-MM-dd"` \(e.g., `"2021-12-31"`\). |
+| `start` | `string` | required | Start date in format `"yyyy-MM-dd"` \(e.g. `"2021-12-24"`\).  |
+| `end` | `string` | required | End date \(excluded\) in format `"yyyy-MM-dd"` \(e.g., `"2021-12-31"`\). This date can't be booked for stay anymore. This date is like a reservation departure. |
 | `releasedDate` | `string` | required | End date in format `"yyyy-MM-dd"` \(e.g., `"2021-12-24"`\). |
 
+> **Dates**: Interval for 1 night (e.g. is represented 2021-12-24/25 is represented as `"start": "2021-12-24", "end": "2021-12-24"`).
```

Comment:
I would prefer to avoid additional explanations, because in future this API will likely go to semi-automated documentation, in which case it won't be possible to add additional explanatory text in the middle of the API Reference.

In any case, unless I'm mistaken, the example is wrong! Since the end date is exclusive, like a reservation departure, then surely for a 1 night stay it should be start "2021-12-24" and end "2021-12-25", or am I mistaken?
```suggestion
```

Reply:
right 🤦